packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-testing/About.tool.mdx

```diff
@@ -159,3 +159,14 @@ it('should hide the caret', () => {
   expect(element.getCssProperty('caretColor').value).to.equal('rgba(0,0,0,0)');
 });
 ```
+
+## Locale Update (Case by Case)
+
+### Terra.setLocale
+Use `Terra.setLocale` to update a terra application's locale on a case by case basis; particularly useful for testing locale changes on a deployed application. This method accepts a string containing the `locale` to update to.
+
+```js
+it('should update an application's locale, () => {
+  Terra.setLocale('en');
```

Comment:
I like this syntax a lot. imo we should expose any other terra application switches the same way if a need arises (theme comes to mind). I don't we will find a reason to expose dispatch custom event itself at all.

We could use setLocale in the terra-dev-site project to eliminate a run.